packages/client/hmi-client/src/page/data-explorer/DataExplorer.vue

```diff
@@ -454,6 +458,23 @@ const toggleDataItemSelected = (dataItem: { item: ResultType; type?: string }) =
 	}
 };
 
+const downloadPDF = async (dataItem: { item: ResultType; type?: string }) => {
+	const doi = dataItem.item?.identifier[0]?.id;
```

Comment:
@YohannParis this is throwing a lint error.  Am I doing this wrong?

Reply:
If this is only for Documents and not Model or Dataset. You can do:

```typescript
import { DocumentType } from '@/types/Document';
...
const downloadPDF = async (dataItem: { item: DocumentType; type?: string }) => {
```